prime-router/src/main/resources/metadata/fhir_transforms/common/use-pid-14-for-email.yml

```diff
@@ -0,0 +1,5 @@
+elements:
+  - name: test-element
```

Comment:
I would rename this to something more descriptive

Reply:
also what is this doing?

Reply:
NVM I see what this is doing now, I would still suggest renaming the element but not a high concern